adventure/internal/handler/grpc/grpc.go

```diff
@@ -0,0 +1,74 @@
+package grpc
```

Comment:
Nice find with the remote procedure call!
looks super similar to a controller with all the CRUD func.